### DISCUSSION TITLE
Distorted and black white wallpaper

### DISCUSSION THREAD

**Discussion by trueloften** [end-4/dots-hyprland#347] [Q&A] [answered]
I looked through all the configs, but couldn't find a way to disable this feature(>badapple).

Wallpaper becomes distorted and turns black and white

**Comment by clsty** [comment]
Didn't one of `>dark` `>light` `>color` fix it?

**Reply by end-4** [reply]
no

**Reply by trueloften** [reply]
No
![IMG_20240323_041642_469.jpg](https://github.com/end-4/dots-hyprland/assets/157816056/d5fdf1e6-e65b-4d97-9663-6d061cef9adb)

**Comment by end-4** [comment]
f07f55e2d665a6683bea29e61d28b40974206b1e
this should fix `>badapple` action
alternatively you can change the color style to Monochrome
![image](https://github.com/end-4/dots-hyprland/assets/97237370/973d987d-75cd-4636-a657-f3e52b7475ca)

**Reply by end-4** [reply]
idk what went wrong with your wallpaper though

**Reply by trueloften** [reply]
Sorry, I guess I described the problem incorrectly. Switching the color scheme works properly, but for some reason all the wallpapers are dark and crooked. Is only swww responsible for the wallpaper or something else?

**Reply by trueloften** [reply, marked as answer]
I found a swww issue with this bug. Thanks for the quick response

https://github.com/LGFae/swww/issues/233
